docs/data-sources/mws_credentials.md

```diff
@@ -0,0 +1,42 @@
+---
+subcategory: "AWS"
+---
+# databricks_mws_credentials Data Source
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../index.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _authentication is not configured for provider_ errors.
```

Comment:
in this context `azurerm_databricks_workspace` isn't relevant

Reply:
addressed in latest commit